x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts

```diff
@@ -46,7 +45,10 @@ spec:
               value: "elastic"
             # The basic authentication password used to connect to Elasticsearch
             - name: ES_PASSWORD
-              value: "changeme"
+              value: ""
+            # The Elasticsearch host to communicate with
+            - name: ES_HOST
```

Comment:
@elastic/fleet team comparing with [main](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts) the ES_HOST is missing

Was done on purpose?
Also the ES_PASSWORD will create a problem if is not changeme? 

(@tetianakravchenko FYI as previous PR was closed)

Reply:
Updated the code and new PR is here https://github.com/elastic/kibana/pull/138713/files